{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "import pandas as pd"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "SUBJECT = \"Molecular biology\"\n",
    "FILE_PATH = \"csv_files/\"+ SUBJECT.lower().replace(\" \",\"_\") + \".csv\"\n",
    "df = pd.read_csv(FILE_PATH)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>CONCEPT</th>\n",
       "      <th>RANK</th>\n",
       "      <th>PARENT</th>\n",
       "      <th>LOCK</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>molecular biology</td>\n",
       "      <td>0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>chromosome</td>\n",
       "      <td>1</td>\n",
       "      <td>molecular biology</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>gene</td>\n",
       "      <td>1</td>\n",
       "      <td>molecular biology</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>DNA polymerase</td>\n",
       "      <td>1</td>\n",
       "      <td>molecular biology</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>helicase</td>\n",
       "      <td>1</td>\n",
       "      <td>molecular biology</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>806</th>\n",
       "      <td>Molecular basis</td>\n",
       "      <td>1</td>\n",
       "      <td>molecular biology</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>807</th>\n",
       "      <td>diseases</td>\n",
       "      <td>1</td>\n",
       "      <td>molecular biology</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>808</th>\n",
       "      <td>drug development</td>\n",
       "      <td>1</td>\n",
       "      <td>molecular biology</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>809</th>\n",
       "      <td>technical terms</td>\n",
       "      <td>1</td>\n",
       "      <td>molecular biology</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>810</th>\n",
       "      <td>extract</td>\n",
       "      <td>1</td>\n",
       "      <td>molecular biology</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>811 rows × 4 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "               CONCEPT  RANK             PARENT  LOCK\n",
       "0    molecular biology     0                NaN     1\n",
       "1           chromosome     1  molecular biology     0\n",
       "2                 gene     1  molecular biology     0\n",
       "3       DNA polymerase     1  molecular biology     0\n",
       "4             helicase     1  molecular biology     0\n",
       "..                 ...   ...                ...   ...\n",
       "806    Molecular basis     1  molecular biology     0\n",
       "807           diseases     1  molecular biology     0\n",
       "808   drug development     1  molecular biology     0\n",
       "809    technical terms     1  molecular biology     0\n",
       "810            extract     1  molecular biology     0\n",
       "\n",
       "[811 rows x 4 columns]"
      ]
     },
     "execution_count": 3,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [],
   "source": [
    "from langchain_openai import OpenAI\n",
    "from langchain.prompts import PromptTemplate\n",
    "from langchain.chains import LLMChain\n",
    "from dotenv import load_dotenv\n",
    "load_dotenv()\n",
    "\n",
    "llm = OpenAI(temperature=0.1)\n",
    "\n",
    "subject = \"molecular biology\"\n",
    "\n",
    "prompt_template_backwards = PromptTemplate(\n",
    "            input_variables=[\"subject\",\"concept\"],\n",
    "            template = 'Give me a list of concepts of {subject} which I need to understand before I can understand \"{concept}\" the technical terms. Give me only the terms.'\n",
    "        )\n",
    "\n",
    "backwards_chain = LLMChain(llm=llm,prompt=prompt_template_backwards,output_key=\"concept_list\")\n",
    "\n",
    "import re"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [],
   "source": [
    "values = df[\"CONCEPT\"].values[1:]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [],
   "source": [
    "file_path = \"backwards_files/\"+ subject.lower().replace(\" \",\"_\") + \".csv\"\n",
    "new_df = pd.DataFrame(columns=[\"CONCEPT\",\"PARENT\"])\n",
    "\n",
    "for concept in values: \n",
    "    response = backwards_chain.invoke({\"concept\":concept,\"subject\":subject})\n",
    "    response = response[\"concept_list\"]\n",
    "    regex_pattern=\"\\s*\\n+\\d+\\.\\s*\"\n",
    "    concept_list = re.split(regex_pattern,response)\n",
    "    concept_list = [item for item in concept_list if item!=\"\"]\n",
    "    for new_concept in concept_list: \n",
    "        new_df.loc[len(new_df)] = {\"PARENT\":concept,\"CONCEPT\":new_concept}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>CONCEPT</th>\n",
       "      <th>PARENT</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>DNA</td>\n",
       "      <td>chromosome</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>Gene</td>\n",
       "      <td>chromosome</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>Nucleotide</td>\n",
       "      <td>chromosome</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>RNA</td>\n",
       "      <td>chromosome</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>Protein</td>\n",
       "      <td>chromosome</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>23660</th>\n",
       "      <td>CRISPR/Cas9</td>\n",
       "      <td>extract</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>23661</th>\n",
       "      <td>RNA interference (RNAi)</td>\n",
       "      <td>extract</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>23662</th>\n",
       "      <td>Gene editing</td>\n",
       "      <td>extract</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>23663</th>\n",
       "      <td>Mutagenesis</td>\n",
       "      <td>extract</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>23664</th>\n",
       "      <td>Bioinformatics</td>\n",
       "      <td>extract</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>23665 rows × 2 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                       CONCEPT      PARENT\n",
       "0                          DNA  chromosome\n",
       "1                         Gene  chromosome\n",
       "2                   Nucleotide  chromosome\n",
       "3                          RNA  chromosome\n",
       "4                      Protein  chromosome\n",
       "...                        ...         ...\n",
       "23660              CRISPR/Cas9     extract\n",
       "23661  RNA interference (RNAi)     extract\n",
       "23662             Gene editing     extract\n",
       "23663              Mutagenesis     extract\n",
       "23664           Bioinformatics     extract\n",
       "\n",
       "[23665 rows x 2 columns]"
      ]
     },
     "execution_count": 8,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "new_df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [],
   "source": [
    "new_df.to_csv(\"backwards_files/molecular_biology.csv\",index=False)"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": ".venv",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.11.3"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
